### PR TITLE
Added content on using Truffle with Pantheon 

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added content on using Truffle with Pantheon

### CODE REVIEW

File: docs/Using-Pantheon/Truffle.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+description: Using Pantheon with Truffle
+<!--- END of page meta data -->
+
+# Using Pantheon with Truffle 
+
+Developing for Pantheon using Truffle is the same as using Truffle to develop for the public Ethereum networks. 
+Truffle supports Pantheon with the only difference being Pantheon does not implement private key management. 
+To use Pantheon with Truffle, you must configure the Truffle wallet.
```
Comment (lucassaldanha) [inline docs/Using-Pantheon/Truffle.md]: I'm not sure if we should call it "THE Truffle wallet" or "A Truffle wallet". I believe there are other wallet implementations for truffle out there.

File: docs/Using-Pantheon/Truffle.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## Install Truffle Wallet 
+
+To install the [Truffle wallet](https://github.com/trufflesuite/truffle-hdwallet-provider):
```
Comment (lucassaldanha) [inline docs/Using-Pantheon/Truffle.md]: This seems to be deprecated now. We might need to update it soon. I believe the instructions will be the same. I'm not sure on what we need to update though.

File: docs/Using-Pantheon/Truffle.md (hunk 1 above)
Comment (lucassaldanha) [reply docs/Using-Pantheon/Truffle.md]: But this seems to be the "official" implementation so it should be fine.

Comment (macfarla) [reply docs/Using-Pantheon/Truffle.md]: I would go with "a Truffle wallet" - there are a few

File: docs/Using-Pantheon/Truffle.md (hunk 2 above)
Comment (macfarla) [reply docs/Using-Pantheon/Truffle.md]: yeah instructions are the same, maybe just remove the github link

Comment (MadelineMurray) [reply docs/Using-Pantheon/Truffle.md]: Done

File: docs/Using-Pantheon/Truffle.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Using-Pantheon/Truffle.md]: Done

File: docs/Using-Pantheon/Truffle.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+```bash
+npm install --save truffle-hdwallet-provider@web3-one
+```
+
+!!!note
+    With Truffle 5, you must use a Web3 1.0 enabled wallet or the Truffle tasks hang.
+
+    `npm` requires `sudo` on Linux.
```
[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/Using-Pantheon/Truffle.md]: Removed

**Pull request merged**